components/molecule/thumbnail/src/index.scss

```diff
@@ -0,0 +1,97 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$sizes-sui-molecule-thumbnail: (
+  large: 144px,
+  medium: 72px,
+  small: 48px,
+  xsmall:  40px
+) !default;
+
+$ratios-sui-molecule-thumbnail: (
+  1: 1,
+  4: 3,
+  16: 9
+) !default;
+
+$bd-outer-sui-molecule-thumbnail: 1px solid color-variation($c-gray, 3) !default;
+$bdc-outer-hover-sui-molecule-thumbnail: $c-text-link !default;
+$bd-inner-sui-molecule-thumbnail: 3px solid $c-white !default;
+$bgc-sui-molecule-thumbnail: $c-gray-light !default; 
+$bs-sui-molecule-thumbnail: 0 !default;
+
+a {
```

Comment:
we dont' usually style elements. We use to style classes.
can somebody confirm if this has changed?

Reply:
Agree

Reply:
Ok. So, i should create an element for each one then?